Holovibes/sources/api/contrast_api.cc

```diff
@@ -13,23 +15,30 @@ float ftruncate(const int precision, float value)
 
 #pragma endregion
 
+#pragma region Log
+
+bool ContrastApi::get_log_enabled() const { return get_log_enabled(api_->view.get_current_window_type()); }
+
+void ContrastApi::set_log_enabled(bool value) const
+{
+    return set_log_enabled(api_->view.get_current_window_type(), value);
+}
+
+#pragma endregion
+
 #pragma region Contrast
 
-float get_contrast_min(WindowKind kind)
+float ContrastApi::get_contrast_min(WindowKind kind) const
 {
     float min = get_window(kind).contrast.min;
     return get_log_enabled(kind) ? min : log10(min);
 }
 
-float get_contrast_max(WindowKind kind)
-{
-    float max = get_window(kind).contrast.max;
-    return get_log_enabled(kind) ? max : log10(max);
-}
+float ContrastApi::get_contrast_min() const { return get_contrast_min(api_->view.get_current_window_type()); }
```

Comment:
You could merge the second function with the first one with a default arg as the call toget_current_window_type